README.md

```diff
@@ -503,37 +270,159 @@ const createI18nWidget = createWidgetBase
+						label: messages.format('itemCount', { count: 2 })
 					})
 				];
 			}
 		}
 	});
 
 const widget = createI18nWidget({
-	state: {
-		// Set the locale for the widget and all of its children. Any child can still
-		// set its own locale.
-		locale: 'fr'
+	// Set the locale for the widget and all of its children. Any child can
+	// still set its own locale.
+	locale: 'fr'
+});
+```
+
+#### Projector
+
+To render a widget into the DOM projector traits need to mixed in using the `createProjectorMixin`. It is possible to have many projectors and attach them to elements in the DOM. However widgets with projector traits must not be nested.
+
+Projectors operate in the same way as any widget except need to be instantiated and managed outside of the usual lifecycle. To attach the projector to the DOM call either `.append`, `.merge` or `.replace` depending on the type of attachment required.
+
+ - append  - Creates the widget as a child to the projector's `root` node
+ - merge   - Merges the widget with the projector's `root` node
+ - replace - Replace the projector's `root` node with the widget
+
+Using the `createProjectorMixin` to turn any widget into a projector:
+
+```ts
+import createProjectorMixin from '@dojo/widgets/mixins/createProjectorMixin';
+import createMyWidget from './createMyWidget';
+
+const myProjectorWidget = createMyWidget.mixin(createProjectorMixin)();
+
+myProjectorWidget.append().then(() => {
+	// appended
+});
+```
+At times it may be required to directly instantiate a dedicated projector using `createProjector` and manually set the children for attaching to the DOM.
+
+```ts
+import { DNode } from '@dojo/widgets/interfaces';
+import { w } from '@dojo/widgets/d';
+import createProjector, { Projector } from '@dojo/widgets/createProjector';
+
+import createButton from './createMyButton';
+import createTextInput from './createMyTextInput';
+
+const projector = createProjector();
+
+projector.setChildren([
+	w(createMyTextInput, { id: 'textinput' }),
+	w(createMyButton, { id: 'button', { label: 'Button' })
+]);
+
+projector.append().then(() => {
+	console.log('projector is attached');
+});
+```
+
+### Authoring Examples
+
+#### Sample Label Widget
+
+A simple widget with no children such as a `label` widget can be created like this:
+
+```ts
+import { VNodeProperties } from '@dojo/interfaces/vdom';
+import { Widget, WidgetFactory, WidgetProperties } from '@dojo/widgets/interfaces';
+import createWidgetBase from '@dojo/widgets/createWidgetBase';
+
+export interface LabelProperties extends WidgetProperties {
+    label: string;
+}
+
+export type Label = Widget<LabelProperties>
+
+export interface LabelFactory extends WidgetFactory<Label, LabelProperties> {}
+
+const createLabelWidget: LabelFactory = createWidgetBase.mixin({
+    mixin: {
+        tagName: 'label',
+        nodeAttributes: [
+            function(this: Label): VNodeProperties {
+                return { innerHTML: this.properties.label };
```

Comment:
Thanks! (You're right, it is what's in there now. Just hadn't noticed it before :P)